src/core/Akka.Streams.Tests/Akka.Streams.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <AssemblyName>Akka.Streams.Tests</AssemblyName>
     <TargetFrameworks>$(NetFrameworkTestVersion);$(NetTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+    <LangVersion>9</LangVersion>
```

Comment:
Added this in order to natively support `IAsyncEnumerable` inside both Akka.Streams and its test suite. We can consider dropping the dependencies on the BCL NuGet package that pulls it in now.

Reply:
```xml
  <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
```

Is referenced in Akka.Streams - let me see if I can remove it in an additional commit.